projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py

```diff
@@ -238,17 +249,18 @@ def deploy(
     # to accumulate desired background livetime
     state_flags = [f"{ifo}:{state_flag}" for ifo in ifos]
     segments = query_segments(state_flags, start, stop, min_segment_length)
-    shifts_required = utils.calc_shifts_required(segments, Tb, max(shifts))
+    shifts_required = utils.get_num_shifts(segments, Tb, shifts[1])
```

Comment:
Was there a reason to switch from `max(shifts)`?

Reply:
No good call that got lost in the shuffle